src/overlays/actors/ovl_En_Mm2/z_en_mm2.h

```diff
@@ -6,9 +6,24 @@
 
 struct EnMm2;
 
+#define SAVE_TIMER_2 ((void)0, gSaveContext.timer2Value)
```

Comment:
I personally don't think we should have a macro for this. All the `(void)0` accesses will be cleaned up eventually so I wouldn't worry about having a few instances of it in the code.